### PR TITLE
new cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

File: calibration/METBKA/CGINS-METCTD-11117__20240118.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+37-11117,CC_a1,3.018077e-004,
+37-11117,CC_a2,-4.177053e-006,
+37-11117,CC_a3,1.944815e-007,
+37-11117,CC_g,-9.987949e-001,
+37-11117,CC_h,1.355772e-001,
+37-11117,CC_i,-1.725624e-004,
+37-11117,CC_j,3.133591e-005,
+37-11117,CC_ctcor,3.250000e-006,
+37-11117,CC_cpcor,-9.570000e-008,
+37-11117,CC_wbotc,4.601398e-007,
```
Comment (reedan88) [inline calibration/METBKA/CGINS-METCTD-11117__20240118.csv:12]: Looks good.

File: calibration/METBKA/CGINS-METCTD-11560__20240105.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+37-11560,CC_a1,3.140875e-004,
+37-11560,CC_a2,-5.055243e-006,
+37-11560,CC_a3,2.198803e-007,
+37-11560,CC_g,-9.760012e-001,
+37-11560,CC_h,1.279654e-001,
+37-11560,CC_i,-3.282179e-004,
+37-11560,CC_j,3.895180e-005,
+37-11560,CC_ctcor,3.250000e-006,
+37-11560,CC_cpcor,-9.570000e-008,
+37-11560,CC_wbotc,9.030768e-008,
```
Comment (reedan88) [inline calibration/METBKA/CGINS-METCTD-11560__20240105.csv:12]: Looks good.

File: calibration/METBKA/CGINS-METCTD-13543__20240105.csv [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+37-13543,CC_a1,3.083857e-004,
+37-13543,CC_a2,-4.596564e-006,
+37-13543,CC_a3,2.048180e-007,
+37-13543,CC_g,-9.926559e-001,
+37-13543,CC_h,1.350159e-001,
+37-13543,CC_i,-2.480443e-004,
+37-13543,CC_j,3.599593e-005,
+37-13543,CC_ctcor,3.250000e-006,
+37-13543,CC_cpcor,-9.570000e-008,
+37-13543,CC_wbotc,3.989240e-007,
```
Comment (reedan88) [inline calibration/METBKA/CGINS-METCTD-13543__20240105.csv:12]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00205__20240109.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+ACS-205,CC_tbins,"[0.866495, 1.279220, 2.440625, 3.452258, 4.458750, 5.492727, 6.530800, 7.493333, 8.496522, 9.490435, 10.498462, 11.513636, 12.481739, 13.492800, 14.486087, 15.468000, 16.467083, 17.471200, 18.488077, 19.486667, 20.477200, 21.488000, 22.493200, 23.482000, 24.469600, 25.482917, 26.470400, 27.485926, 28.500800, 29.498077, 30.499200, 31.495556, 32.503571, 33.505313, 34.467742]",
+ACS-205,CC_taarray,SheetRef:CC_taarray,
+ACS-205,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00205__20240109.csv:9]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00205__20240109__CC_taarray.ext [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+-0.006165,-0.005495,-0.004132,-0.004250,-0.003799,-0.003529,-0.003349,-0.003071,-0.003025,-0.002876,-0.002454,-0.002294,-0.001959,-0.001721,-0.001504,-0.001910,-0.001845,-0.001491,-0.001034,-0.000934,-0.000868,-0.000497,-0.000239,-0.000297,-0.000438,0.000000,0.000158,0.000254,0.000336,0.000411,0.000538,0.000711,0.000589,0.000590,0.000640
+-0.006212,-0.005758,-0.004354,-0.004494,-0.004068,-0.003802,-0.003567,-0.003277,-0.003226,-0.003043,-0.002575,-0.002421,-0.002161,-0.001848,-0.001650,-0.002047,-0.001988,-0.001494,-0.001083,-0.001029,-0.000923,-0.000445,-0.000152,-0.000287,-0.000389,0.000000,0.000210,0.000284,0.000314,0.000500,0.000709,0.000833,0.000501,0.000664,0.000795
+-0.008076,-0.007341,-0.005505,-0.005717,-0.005016,-0.004651,-0.004332,-0.003852,-0.003988,-0.003799,-0.003245,-0.002987,-0.002173,-0.002101,-0.001710,-0.002562,-0.002403,-0.001960,-0.001269,-0.001199,-0.001073,-0.000433,0.000022,-0.000317,-0.000489,0.000000,0.000234,0.000459,0.000245,0.000657,0.000796,0.000909,0.000607,0.000533,0.000818
```
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00205__20240109__CC_taarray.ext:90]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00205__20240109__CC_tcarray.ext [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+0.000476,0.001747,0.002925,0.002919,0.002670,0.002489,0.002309,0.002413,0.002241,0.002210,0.002042,0.001758,0.001544,0.001355,0.001199,0.001010,0.000961,0.001031,0.001077,0.000944,0.000622,0.000467,0.000535,0.000354,0.000255,0.000000,-0.000005,0.000018,-0.000321,-0.000378,-0.000681,-0.000755,-0.000920,-0.001336,-0.001384
+0.000089,0.001997,0.002676,0.002550,0.002290,0.001992,0.001730,0.001993,0.001904,0.002016,0.001852,0.001731,0.001546,0.001222,0.001511,0.001224,0.000901,0.000934,0.000895,0.000833,0.000438,0.000446,0.000325,0.000448,0.000071,0.000000,-0.000385,-0.000557,-0.000339,-0.000577,-0.000935,-0.000957,-0.001262,-0.001646,-0.001639
+-0.000020,0.000846,0.001769,0.001895,0.001626,0.001603,0.001487,0.001826,0.001555,0.001856,0.001654,0.001512,0.001363,0.001312,0.001300,0.001068,0.000911,0.001024,0.000991,0.000976,0.000633,0.000542,0.000482,0.000392,0.000218,0.000000,-0.000152,-0.000416,-0.000688,-0.000558,-0.000932,-0.001223,-0.001674,-0.001920,-0.001964
```
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00205__20240109__CC_tcarray.ext:90]: Looks good.

File: calibration/PCO2WB/CGINS-PCO2WB-C0134__20240123.csv [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+C0134,CC_cala,0.0019,source_file PCO2W-B_SAMI-CO2_SN_C0134_Calibration_Files_2024-01-23.zip > C0134_SAMI_230124_CalCert.pdf
+C0134,CC_calb,0.9209,
+C0134,CC_calc,-2.2736,
+C0134,CC_calt,8.5293,
+C0134,CC_ea434,19706,constant
+C0134,CC_ea620,34,constant
+C0134,CC_eb434,3073,constant
+C0134,CC_eb620,44327,constant
+C0134,CC_sami_bits,12,
+C0134,CC_cal_range,"[212,993]",
```
Comment (reedan88) [inline calibration/PCO2WB/CGINS-PCO2WB-C0134__20240123.csv:11]: Looks good.

File: calibration/PCO2WC/CGINS-PCO2WC-C0107__20240123.csv [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+C0107,CC_cala,-0.0345, source_file PCO2W-C_SAMI-CO2_SN_C0107_Calibration_Files_2024-01-23.zip > C0107SAMI_230124_CalCert.pdf
+C0107,CC_calb,1.1488,
+C0107,CC_calc,-2.6180,
+C0107,CC_calt,8.7148,
+C0107,CC_ea434,19706,constant
+C0107,CC_ea620,34,constant
+C0107,CC_eb434,3073,constant
+C0107,CC_eb620,44327,constant
+C0107,CC_sami_bits,14,
+C0107,CC_cal_range,"[202,989]",
```
Comment (reedan88) [inline calibration/PCO2WC/CGINS-PCO2WC-C0107__20240123.csv:11]: Looks good.

File: calibration/PCO2WC/CGINS-PCO2WC-C0140__20240123.csv [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+C0140,CC_cala,-0.0116,source_file PCO2W-C_SAMI-CO2_SN_C0140_Calibration_Files_2024-01-23.zip > C0140_SAMI_230124_CalCert.pdf
+C0140,CC_calb,1.0138,
+C0140,CC_calc,-2.4285,
+C0140,CC_calt,8.5465,
+C0140,CC_ea434,19706,constant
+C0140,CC_ea620,34,constant
+C0140,CC_eb434,3073,constant
+C0140,CC_eb620,44327,constant
+C0140,CC_sami_bits,14,
+C0140,CC_cal_range,"[233,990]",
```
Comment (reedan88) [inline calibration/PCO2WC/CGINS-PCO2WC-C0140__20240123.csv:11]: Looks good.

File: calibration/PHSENF/CGINS-PHSENF-P0079__20240105.csv [hunk 10]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0079,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0079_Validation_Files_2024-01-05.zip > SAMI_P0079_050124t_ValCert.pdf
+P0079,CC_ea578,94.1,
+P0079,CC_eb434,2284.0,
+P0079,CC_eb578,38676.0,
+P0079,CC_ind_off,0,
+P0079,CC_ind_slp,1,
+P0079,CC_psal,35,Constant
+P0079,CC_sami_bits,12,
```
Comment (reedan88) [inline calibration/PHSENF/CGINS-PHSENF-P0079__20240105.csv:9]: Looks good.

File: calibration/PHSENF/CGINS-PHSENF-P0104__20240105.csv [hunk 11]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0104,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0104_Validation_Files_2024-01-05.zip > SAMI_P0104_050124t_ValCert.pdf
+P0104,CC_ea578,94.1,
+P0104,CC_eb434,2284.0,
+P0104,CC_eb578,38676.0,
+P0104,CC_ind_off,0,
+P0104,CC_ind_slp,1,
+P0104,CC_psal,35,Constant
+P0104,CC_sami_bits,12,
```
Comment (reedan88) [inline calibration/PHSENF/CGINS-PHSENF-P0104__20240105.csv:9]: Looks good.

File: calibration/PHSENF/CGINS-PHSENF-P0147__20240105.csv [hunk 12]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0147,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0147_Validation_Files_2024-01-05.zip > SAMI_P0147_050124t_ValCert.pdf
+P0147,CC_ea578,94.1,
+P0147,CC_eb434,2284.0,
+P0147,CC_eb578,38676.0,
+P0147,CC_ind_off,0,
+P0147,CC_ind_slp,1,
+P0147,CC_psal,35,Constant
+P0147,CC_sami_bits,12,
```
Comment (reedan88) [inline calibration/PHSENF/CGINS-PHSENF-P0147__20240105.csv:9]: Looks good.

File: calibration/FLORTD/CGINS-FLORTD-01300__20240108.csv [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+1300,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1300_Calibration_Files_2024-01-08.zip > BBFL2W-1300.dev
+1300,CC_dark_counts_cdom,43,
+1300,CC_dark_counts_chlorophyll_a,59,
+1300,CC_dark_counts_volume_scatter,60.025,
+1300,CC_depolarization_ratio,0.039,Constant
+1300,CC_measurement_wavelength,700,Constant
+1300,CC_scale_factor_cdom,9.058E-02,
+1300,CC_scale_factor_chlorophyll_a,1.221E-02,
+1300,CC_scale_factor_volume_scatter,1.914E-06,
+1300,CC_scattering_angle,124,Constant
```
Comment (reedan88) [inline calibration/FLORTD/CGINS-FLORTD-01300__20240108.csv:11]: Looks good!

**Review by reedan88 — APPROVED** [review]
Update looks good.

**Review by nichole-arm — APPROVED** [review]
all looks good

**Pull request merged**